CHANGES_UNRELEASED.md

```diff
@@ -2,4 +2,13 @@
 
 # Unreleased Changes
 
-[Full Changelog](https://github.com/mozilla/application-services/compare/v0.46.0...master)
\ No newline at end of file
+[Full Changelog](https://github.com/mozilla/application-services/compare/v0.46.0...master)
+
+
+## Logins
+
+### What's new
```

Comment:
I wonder if it's worth expanding here to say something like "because it's semantics have already diverged" or similar, to help convey some degree of urgency? Your call!

Reply:
Good idea, added 👍